backend/coins/eth/coin_test.go

```diff
@@ -46,7 +46,7 @@ func (s *testSuite) SetupTest() {
 
 	s.ERC20Coin = NewCoin(
 		nil,
-		coin.CodeERC20TEST,
+		"ERC20TEST",
```

Comment:
In other places you removed something about ERC20Test... is this still used?

Reply:
It is for unit testing erc20 token stuff. The name here doesn't really matter, same as the fake contract address below it.